iXBRLViewerPlugin/viewer/src/js/ixnode.js

```diff
@@ -32,6 +32,7 @@ export function IXNode(id, wrapperNode, docIndex) {
     this.docIndex = docIndex;
     this.footnote = false;
     this.id = id;
+    this.visible = true;
```

Comment:
I assume "hidden" was not used to avoid concusion with ix:hiddren, but "visible" seems to imply CSS "visibility".  Perhaps `htmlHidden` would be clearer?

Perhaps the CSS class should similarly be `html-hidden-fact-tag` for consistency?

Reply:
Good call - done in 88edf9d